docs/reference/modules/piece-page-type.md

```diff
@@ -93,7 +99,7 @@ These include:
     - string
     - url
 
-When the index page is served, configured filters will be represented on a `req.data.piecesFilter` object (`data.piecesFilter` in the template). If you include `counts: true` in a filter object the number of pieces matching that filter are included in the `req.data.piecesFilter` properties.
+When the index page is served, filter data will be returned in the `req.data.piecesFilters` object (`data.piecesFilters` in the template). This object consists of an array for each configured filter. That array contains objects with `value` and `label` properties for every `piece-type` that matches the filter. If `counts: true` is included for the filter query, each object in the array will also have a `count` property with the number of matching pieces.
```

Comment:
Passing any of these filters back to the index page as query string parameters will filter the results accordingly. <-- This is the actual win of having the filters, so we should spell it out. An example using `| build` would be nice, but maybe more appropriate for a new "guide" advanced page about these features, which currently are documented only here and in the "module options" reference page .

Reply:
Changed. I should have thought to include this. It was one of my 'AhhHaa!' moments working on this documentation. I didn't know this fact and suddenly discovered this fact and a LOT of things gelled in my head. Not sure if you want the `| build` example here or not - up to you.